ansible_collections/arista/avd/roles/eos_designs/tasks/main.yml

```diff
@@ -11,6 +11,20 @@
   run_once: true
   failed_when: false
 
+- name: "Verify Requirements"
+  tags: [debug, avd_req]
+  arista.avd.verify_requirements:
+    dependencies: "{{ lookup('file', requirements_path ~ '/requirements.txt').splitlines() }}"
+  vars:
+    requirements_path: "{{ (role_path | split('/'))[0:-2] | join('/') }}"
+  run_once: true
+  register: requirements
+
+- name: "Print requirements"
```

Comment:
please make the tags for this task `debug, avd_req, never`

Reply:
MAde it for both tasks

Reply:
well we don't want `never` on the verify requirements.